### DISCUSSION TITLE
YouTube Studio MicroG

### DISCUSSION THREAD

**Discussion by t0ry003** [ReVanced/revanced-discussions#232] [Question]
## Problem
> I am locked from using YT Studio because I have no gapps support on my Huawei device.

## Suggestion
> Add an YouTube Studio patch for MicroG. I am sure that some people including me will find it very helpful.

#### This suggestion started on Reddit at [youtube_studio_question](https://www.reddit.com/r/revancedapp/comments/vctxmo/youtube_studio_question/)

**Comment by DraakjeYoblama** [comment]
I don't think this is something the Revanced team will do as it has almost nothing to do with the existing project

You could always use the youtube studio website.

**Reply by t0ry003** [reply]
yes ofc! can u show me the yt studio mobile interface? It's optimized just for desktop use. But fine...

**Comment by nyxiereal** [comment]
You can use vanced MicroG, works for me

**Reply by t0ry003** [reply]
how? on YouTube studio?

**Reply by nyxiereal** [reply]
ReVanced have no support to yt studio, sorry

**Reply by t0ry003** [reply]
@xemulat I knew. Don't worry 👍🏻 :)

**Comment by TheJeterLP** [comment]
This has to be implemented by microG, not ReVanced.

**Reply by aidanmacgregor** [reply]
I thought this would be a revanced issue, with YouTube as I understand they modify it to point to micro g instead of proper GMS, a patch to studio to point to micro g I think would be nesicerry to make this work
